src/helpers/getRandomNumberFromUUID.js

```diff
@@ -0,0 +1,13 @@
+const { v4: uuidv4 } = require('uuid');
+
+function getRandomNumberFromUUID() {
+  const uuidBuffer = uuidv4(null, Buffer.allocUnsafe(16));
+
+  const randomNumber = uuidBuffer.readUInt32BE(0);
+
+  return randomNumber;
+}
+
+module.exports = {
+  getRandomNumberFromUUID,
+};
```

Comment:
You don't need to generate id manually, database can do it itself if you set default value to column or set to autoincrement